cinema/views.py

```diff
@@ -197,3 +198,26 @@ def get_serializer_class(self):
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "items": {"type": "string"}},
+                description="Filter by title(example: ?title=inception)",
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genres id(example: ?genres=2,3)",
+            ),
+            OpenApiParameter(
+                "actors",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by actors id(example: ?actors=4,1)",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
```

Comment:
are u sure that order can be filtered by genres,actors,title ?
maybe it is bettert to add docs for movie,moviesession views ?